server/protocols/jmap-rfc8621-integration-tests/pom.xml

```diff
@@ -60,6 +61,12 @@
                 <artifactId>jmap-rfc8621-integration-tests-common</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>james-server-jmap-draft-integration-testing</artifactId>
```

Comment:
We should not depend on draft something. Please duplicate the annotation.

Reply:
Junit 5 change from `Category` to `Tag` and it just need tag's name, we dont need create interface anymore.
We just need remove it and duplicate nothing.